exercises/markdown/README.md

```diff
@@ -0,0 +1,29 @@
+# Markdown
+
+Refactor a Markdown parser.
+
+The markdown exercise is a refactoring exercise. There is code that parses a given string with Markdown syntax and returns the associated HTML for that string. Even though this code is confusingly written and hard to follow, somehow it works and all the tests are passing! Your challenge is to re-write this code to make it easier to read and maintain while still making sure that all the tests keep passing.
+
```

Comment:
It looks like you might not have used the 'configlet' to generate the README.md. If you need a hint about this, use
```
./bin/fetch-configlet
./bin/configlet --help
./bin/configlet generate --help
```